worrying warning for InvertAxis

VS 2015 gives
```
c:\projects\stir\src\spatial_transformation_buildblock\invertaxis.cxx(90): warning C4715: 'stir::InvertAxis::invert_axis_index': not all control paths return a value [C:\projects\stir\build\src\spatial_transformation_buildblock\spatial_transformation_buildblock.vcxproj]```
@danieldeidda can you  have a look?

--- a/src/include/stir/spatial_transformation/InvertAxis.h
+++ b/src/include/stir/spatial_transformation/InvertAxis.h
@@ -30,10 +30,19 @@
 
 START_NAMESPACE_STIR
 
+/*! 
+  \ingroup spatial_transformation
+ 
+  \brief a utility class to "invert" an axis
+  \warning this will reorder the voxel values without adjusting the geometric information.
+*/
 class InvertAxis{
 
 public:
 
+  //! transform the image
+  /*! \a axis_name has to be x, y, z. Otherwise error() will be called.
+   */
 void
 invert_axis(DiscretisedDensity<3, float> &inverted_image,
            const DiscretisedDensity<3, float> &input_image,

--- a/src/spatial_transformation_buildblock/InvertAxis.cxx
+++ b/src/spatial_transformation_buildblock/InvertAxis.cxx
@@ -86,6 +86,12 @@ InvertAxis::invert_axis_index(const int input_index,
 
     else if (axis_name=="z")
         return size -1 -input_index;
+    else
+      {
+        error("InvertAxis: invalid axis name: " + axis_name);
+        return 0; // to avoid compiler warning
+      }
+    
 
 }
 END_NAMESPACE_STIR

--- a/src/utilities/invert_axis.cxx
+++ b/src/utilities/invert_axis.cxx
@@ -18,7 +18,7 @@
  \file
  \ingroup utilities
 
- \brief This program inverts x y or z axis. It is also able to remove nan from the image.
+ \brief This program inverts x y or z axis.
  \author Daniel Deidda
  */
 #include "stir/DiscretisedDensity.h"
@@ -33,7 +33,8 @@ int main(int argc, char **argv)
   if(argc!=4) {
     std::cerr << "Usage: " << argv[0]
               << " <axis name>  <output filename> <input filename>\n"
-              << " if <axis name>= nan the utility will substitute every nan value with zero  ";
+              << " <axis name> has to be x, y, z\n"
+              << "\nWARNING: this will reorder the voxel values without adjusting the geometric information\n";
     exit(EXIT_FAILURE);
   }
   InvertAxis invert;

